src/solidity/types/Enum.js

```diff
@@ -13,7 +14,9 @@ function Enum (enumDef) {
 }
 
 Enum.prototype.decodeFromStorage = function (location, storageContent) {
-  return '<not implemented yet>'
+  var value = util.extractValue(location, storageContent, this.storageBytes)
+  value = parseInt(value)
+  return this.enumDef.children[value].attributes.name
```

Comment:
Please check if it is bounds and return something like `INVALID_ENUM(<value>)` if not